### ISSUE TITLE
Missing content

### ISSUE THREAD

**Issue by cedric-joaquin** [learn-co-curriculum/positioning-elements-with-css#4]
In the lesson, it says: 

_An example of using float for layout structure can be seen in the snippet below:_

And there is no example following it.

**Comment by Sdcrouse** [comment]
Yeah, as far as I can tell, the code snippets are all in `<iframe>`s, and something about them is conflicting with Markdown and/or the Learn app. I have no idea how to fix this (I'm a student, too), but I did find a way to make the code snippets visible (just be sure not to refresh the lesson after you do this):

1. Go to the lesson's README at https://github.com/learn-co-curriculum/positioning-elements-with-css/blob/master/README.md
2. In the top-right corner, click the "Raw" button. You'll be able to see the raw Markdown for this lesson.
3. Each time you see an `<iframe>`, copy it and go back to the Learn lesson.
4. Right-click on the paragraph right above where that `<iframe>` should be and select "Inspect".
5. The `<p>` tag for that paragraph should be selected. Now, right-click on it and select "Edit as HTML".
6. Next to the closing `<p>` tag, paste in the `<iframe>`.
7. Click just to the left of that `<p>` tag without going outside of the Inspector. The `<iframe>` should now show up on the page properly.
8. Lastly, repeat Steps 3-7 for the other `<iframe>` elements.

Hope that somewhat helps!
